resources/js/electron-plugin/src/server/php.ts

```diff
@@ -295,8 +295,8 @@ function getDefaultEnvironmentVariables(secret?: string, apiPort?: number): Envi
         NATIVEPHP_VIDEOS_PATH: getPath('videos'),
         NATIVEPHP_RECENT_PATH: getPath('recent'),
         NATIVEPHP_EXTRAS_PATH: app.isPackaged
-            ? join(app.getAppPath(), 'nativephp', 'extras')
-            : join(app.getAppPath(), '..', '..', 'extras'),
+            ? join(process.resourcesPath, '..', 'extras')
+            : join(process.env.APP_PATH, 'extras'),
```

Comment:
I installed the electron package from a symlinked directory. 
The original `join(app.getAppPath(), '..', '..', 'extras')` pointed to a wrong relative path

Using `APP_PATH` makes this consistent no matter how you structure your dev setup